test/scenarios/cats/validation/DefaultValuesOfCorrectType.yaml

```diff
@@ -85,5 +85,5 @@ tests:
       validate: [DefaultValuesOfCorrectType]
     then:
       - error-count: 1
-      - error: "Variable '$a' of type 'ComplexInput' has invalid default value: {intField: 3}. Reason: The NotNull field 'requiredField' defined in the input type 'Boolean' is missing."
-        loc: {line: 1, column: 47}
\ No newline at end of file
+      - error: "Variable '$a' of type 'ComplexInput' has invalid default value: {intField: 3}. Reason: The NotNull field 'requiredField' of type 'Boolean' is missing."
```

Comment:
Technically, this is a deviation from the CAT, but right now I can't distinguish an input type from a non-input type and it doesn't seem important.